### PR TITLE
feat: Update pyvolt-dpsim-demo instructions to clarify

### PR DESCRIPTION
**Pull request by leblowl**
Signed-off-by: Lucas Leblow <lucasleblow@mailbox.org>

### CODE REVIEW

File: pyvolt-dpsim-demo/README.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Preliminaries
 
+Follow the instructions here to get started:
+https://sogno-platform.github.io/docs/getting-started/
```
Comment (leblowl) [inline pyvolt-dpsim-demo/README.md:6]: I did not have Kubernetes installed and am pretty unfamiliar with the ecosystem, so I was confused with the instructions until I followed the getting started guide.

File: pyvolt-dpsim-demo/README.md [hunk 2]
```diff
@@ -54,6 +57,9 @@ HugePages_Surp:        0
 Hugepagesize:       2048 kB
 Hugetlb:         2097152 kB
 
+If you don't see 1024 next to HugePages_Total, you may need to restart
+your system and try again with a fresh boot.
```
Comment (leblowl) [inline pyvolt-dpsim-demo/README.md:61]: I noticed on my system, that I would issue the command to set HugePages config and it wouldn't set 1024, but something less instead:
```
  lucas@debian:~/src/sogno-platform/example-deployments/pyvolt-dpsim-demo$ echo 1024 | sudo tee /proc/sys/vm/nr_hugepages
  1024
  lucas@debian:~/src/sogno-platform/example-deployments/pyvolt-dpsim-demo$ cat /proc/meminfo | grep Huge
  AnonHugePages:    430080 kB
  ShmemHugePages:        0 kB
  FileHugePages:         0 kB
  HugePages_Total:     117
  HugePages_Free:      117
  HugePages_Rsvd:        0
  HugePages_Surp:        0
  Hugepagesize:       2048 kB
  Hugetlb:          239616 kB
```
I assume this was due to my limited system resources and the programs I was running. When I restarted and issued the command on a fresh boot, it succeeded. This happened on two systems I have.

Also, should we recommend that people persist this setting or at least remind them that it's not persistent?

File: pyvolt-dpsim-demo/README.md [hunk 3]
```diff
@@ -94,11 +100,11 @@ The configmap contains a demo dashboard and should automatically be recognized b
 
 ### CIM Editor Pintura
 
-The following installation will deploy a Pintura instance that is available at the nodePort specified in the pintura_values.yaml file. 
-Per default at port 31234: http://localhost:31234/
+The following installation will deploy a Pintura instance that is available at the nodePort specified in the pintura_values.yaml file.
+Per default at port 31234: http://localhost:31234/ (it may take a few moments to launch and become available)
```
Comment (leblowl) [inline pyvolt-dpsim-demo/README.md:104]: I thought it didn't work when I ran the command and immediately clicked on the link, but it just needed some time.

**Comment by leblowl** [comment]
@m-mirz Hey! I reviewed these docs as you requested, they mostly just worked and I edited them when I ran into issues and provided some comments in this MR to explain and ask further questions. I hope this is in-line with what you were thinking. I have never used Kubernetes before and my laptop struggled to run everything and a browser. Desktop did much better. Both have 8GB mem. Could be that I need to tune some OS settings.

One other question I have: Should we add some further documentation that explains what to do with the demo? I setup the infrastructure, but then I am wondering how I use that infrastructure?

File: pyvolt-dpsim-demo/README.md (hunk 2 above)
Comment (m-mirz) [reply pyvolt-dpsim-demo/README.md:61]: The restart is a good advice. Adding a remark that this is not persistent is a good idea as well.

With the new versions of DPsim and VILLASnode we should be able to remove this requirement.

File: pyvolt-dpsim-demo/README.md (hunk 1 above)
Comment (m-mirz) [reply pyvolt-dpsim-demo/README.md:6]: Good point

**Pull request merged**